content/docs/2.13/authentication-providers/aws.md

```diff
@@ -0,0 +1,140 @@
++++
+title = "Pod Identity Webhook for AWS by using IAM Roles for Service Accounts (IRSA)"
++++
+
+[**AWS IAM Roles for Service Accounts (IRSA) Pod Identity Webhook**](https://github.com/aws/amazon-eks-pod-identity-webhook) ([details](https://aws.amazon.com/blogs/opensource/introducing-fine-grained-iam-roles-service-accounts/)) allows you to provide the role name using an annotation on a service account associated with your pod.
+
+You can tell KEDA to use AWS Pod Identity Webhook via `podIdentity.provider`.
```

Comment:
Might be worth adding a line on where should users set this (which resource).

Reply:
I don't get your point, this is an authentication, so TriggerAuthentication is the place. I mean, we aren't doing it either for other auths